lib/wise_homex/models/unsuccessful_visit.ex

```diff
@@ -0,0 +1,13 @@
+defmodule WiseHomex.UnsuccessfulVisit do
+  @moduledoc false
+
+  use WiseHomex.BaseModel
+
+  embedded_schema do
+    belongs_to :household, WiseHomex.Household
+    field :attempt_at, :utc_datetime
+    field :billable, :boolean, default: true
```

Comment:
I don't think we should add `default` options in our WiseHomex models, since that could add confusion. We just want to get the field value directly from Legolas without defaults.

Reply:
Ah. That makes perfect sense. Thank you. Fixed. :sunny: